pekko-kryo-serialization/src/main/resources/reference.conf

```diff
@@ -1,247 +1,19 @@
 #########################################################
-# Akka akka-kryo-serializer Reference Config File #
+# pekko-kryo-serializer Reference Config File #
 #########################################################
 
 # This is the reference config file that contains all the default settings.
 # Make your edits/overrides in your application.conf.
+# Don't use scala-kryo-serialization to override pekko serialization keys.
```

Comment:
`Don't use scala-kryo-serialization to override pekko serialization keys.` -> maybe better?
`Don't use scala-kryo-serialization.keyX to override pekko serialization keys - use pekko-kryo-serialization.keyX`